syntax = "proto3";

package crawler;

import "google/protobuf/empty.proto";

service CrawlerService {
  rpc AddTarget (AddTargetRequest) returns (google.protobuf.Empty);
}

message AddTargetRequest {
  string target_url = 1;
}
